src/components/text-ellipsis/TextEllipsis.tsx

```diff
@@ -6,6 +6,9 @@ interface TextEllipsisProps extends Omit<React.HTMLProps<HTMLDivElement>, 'type'
     /** Text that should be ellipsed */
     children?: React.ReactNode;
 
+    /** Optional CSS class name */
+    className?: string;
```

Comment:
thought there would come from the extended props no ?

Reply:
Hm, yes, I've made a mistake.

Removed the prop but kept the part where I put it inside the className={ classnames(...) } property.